packages/components/pages/f-offers/package.json

```diff
@@ -66,7 +67,7 @@
   "devDependencies": {
     "@braze/web-sdk": "3.3.0",
     "@justeat/f-button": "4.x",
-    "@justeat/f-content-cards": "8.0.0-beta.4",
```

Comment:
losing this specific version here could be dangerous as people may not see that its running on beta, create v8 of content cards and miss that there is a beta version running and this would auto update to that? then rolling back to older version?

Reply:
Updated to explicitly state version of content cards + made sure it's tagged as "beta"